src/hmem_cuda.c

```diff
@@ -105,11 +150,60 @@ int cuda_copy_from_dev(void *host, const void *dev, size_t size)
 
 int cuda_hmem_init(void)
 {
+#if defined(ENABLE_CUDA_DLOPEN) && defined(HAVE_LIBDL)
```

Comment:
nit: Doesn't ENABLE_CUDA_DLOPEN being set require that HAVE_LIBDL is true?

Reply:
Yeah. Should be with just ENABLE_CUDA_DLOPEN.